safehttp/plugins/xsrf/xsrf.go

```diff
@@ -33,42 +41,71 @@ type UserIdentifier interface {
 	UserID(*safehttp.IncomingRequest) (string, error)
 }
 
-// Interceptor implements XSRF protection. It requires an application key and a
-// storage service. The appKey uniquely identifies each registered service and
-// should have high entropy. The storage service supports retrieving ID's of the
-// application's users. Both the appKey and user ID are used in the XSRF
-// token generation algorithm.
+// Interceptor implements XSRF protection.
 type Interceptor struct {
-	AppKey     string
+	// AppKey uniquely identifies each registered service and should have high
+	// entropy as it is used for generating the XSRF token.
```

Comment:
I would say that this should be secret, and to do that I would rename the field to SecretAppKey.